File: basic.js
function lascii(start, end) {
    const result = [];
    for (let i = start.charCodeAt(0); i <= end.charCodeAt(0); i++) {
        result.push(String.fromCharCode(i));
    }
    return result;
}

// Tests
console.log(lascii('a', 'd'));  // ['a', 'b', 'c', 'd']
console.log(lascii('c', 'i'));  // ['c', 'd', 'e', 'f', 'g', 'h', 'i']
console.log(lascii('t', 'z'));  // ['t', 'u', 'v', 'w', 'x', 'y', 'z']

File: basic.py
def lascii(start, end):
    return [chr(i) for i in range(ord(start), ord(end) + 1)]

# Tests
print(lascii('a', 'd'))   # ['a', 'b', 'c', 'd']
print(lascii('c', 'i'))   # ['c', 'd', 'e', 'f', 'g', 'h', 'i']
print(lascii('t', 'z'))   # ['t', 'u', 'v', 'w', 'x', 'y', 'z']
